client/textBox.py

```diff
@@ -0,0 +1,35 @@
+import pygame as pg
+from definitions import *
+
+class TextBox(pg.Rect):
+    def __init__(self, x, y, w, h, matcher = None):
+        pg.Rect.__init__(self, x, y, w, h)
+        self.color_inactive = BANANA
```

Comment:
the inactive/active logic here is really hard to reason with, because you already have a variable called active, then you have this other variable that mirrors this state, this will be very hard to maintain. Instead, when the rectangle around the textbox is drawn, just have a ternary statement that chooses one color over the other depending on the value of self.active